source/_components/tahoma.markdown

```diff
@@ -0,0 +1,32 @@
+---
+layout: page
+title: "Tahoma"
+description: "Instructions on how to integrate Somfy Tahoma devices with Home Assistant."
+date: 2017-07-18 12:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: somfy.png
+ha_category: Hub
+ha_release: 0.41
+---
+
+
+The `Tahoma` component platform is used as an interface to the [tahomalink.com](https://www.tahomalink.com) website. It adds actually covers and the sun sensor from tahoma plattform.
```

Comment:
Typo: `plattform` -> `platform`.

The last sentence is not quite right. I would suggest removing it or maybe reword it?
Suggestion: `It adds covers and the sun sensor from tahoma platform.`